qiskit_nature/problems/second_quantization/lattice/lattices/lattice.py

```diff
@@ -110,20 +113,27 @@ class Lattice:
     def __init__(self, graph: PyGraph) -> None:
         """
         Args:
-            graph: Input graph for Lattice. `graph.multigraph` must be False.
+            graph: Input graph for Lattice.
```

Comment:
Why this change?

Reply:
I do see that this gets updated further in #472 but that does not mean we can remove this information in this PR, which will potentially be included in `0.3.1` whereas the other PR can only go into `0.4.0`.

Reply:
Whoops, yes it seems I didn't split the PR properly 😉